### PR TITLE
Updated latest Kubernetes version to be set via a variable

### PR DESCRIPTION
**Pull request by idvoretskyi**
Added method to set a Kubernetes variable to the latest actual version.

Updated command with added variable.

Following the https://github.com/kubernetes/kubernetes.github.io/pull/43

### CODE REVIEW

File: docs/getting-started-guides/docker.md [hunk 1]
```diff
@@ -14,7 +14,11 @@ Here's a diagram of what the final result will look like:
 
 1. You need to have docker installed on one machine.
 2. Decide what Kubernetes version to use.  Set the `${K8S_VERSION}` variable to
-   a released version of Kubernetes >= "1.2.0"
+   a released version of Kubernetes >= "v1.2.0". If you'd like to use the latest released version of Kubernetes, run the following:
+
+```sh
+export K8S_VERSION=$(curl -sS https://storage.googleapis.com/kubernetes-release/release/stable.txt)
```
Comment (luxas) [inline docs/getting-started-guides/docker.md]: Can you add the same thing for latest so folks may choose?

Comment (idvoretskyi) [inline docs/getting-started-guides/docker.md]: @luxas what do you mean, sorry?

Comment (luxas) [inline docs/getting-started-guides/docker.md]: Also add
`export K8S_VERSION=$(curl -sS https://storage.googleapis.com/kubernetes-release/release/latest.txt)`
for those who wonder how to get latest

Comment (idvoretskyi) [inline docs/getting-started-guides/docker.md]: @luxas fixed. Thank you.

**Comment by johndmulhausen** [comment]
Thanks, @idvoretskyi !

**Pull request merged**